src/components/Geogebra/GeogebraMergedAdminView.js

```diff
@@ -0,0 +1,452 @@
+import feathersClient from '../../feathers-client';
```

Comment:
This code looks copied over, where is it from?

Reply:
It is taken from one of first versions of student's client or listener